_posts/2013-01-16-future-proof-your-accessibility.md

```diff
@@ -1,19 +1,49 @@
 
-1. [Parsing](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html) relates to ensuring that user agents and AT are able to read and navigate your web site correctly. Validating your site using the [W3C Markup Validator](https://validator.w3.org/nu/) can help you find errors in your markup and ease the burden of the AT parsing your web page. Follow the official specifications for your markup. E.g. make sure to close open tags (`<div class="a-class">...</div>`) and use unique IDs.
-1. [Name, role, value](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html) are related to interface components and controls (i.e, focus states). By default, all standard HTML controls already meet this criteria. This guideline is aimed mainly at developers that design and/or script their own interface components and controls.
+Your future self, coworkers, and most importantly customers and clients will thank you for developing with accessibility in mind, now.
+
+
+## Tech deb goes down when accessibility goes up
+
+The best way to future proof your accessibility efforts is to ensure websites and applications are built with strong semantic foundations. By following just two of [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG21/#robust)' (WCAG) criteria, **Parsing** and **Name, Role, Value**, we can help ensure we are future-proofing our project from tech debt and bugs.
+
+Following these criteria has the bonus value of better supporting current and future [user agents](https://developer.mozilla.org/en-US/docs/Glossary/User_agent), and ensuring assistive technology (AT), such as screen readers, will be more likely to correctly interpret the content of our user interfaces (UI).
+
+### Breaking down the criteria
+
+* [4.1.1 Parsing](https://www.w3.org/TR/WCAG21/#parsing) relates to ensuring that user agents and AT are able to correctly read and navigate your website without complications from erroneous markup.  
+  - Using the [W3C Nu HTML Checker](https://validator.w3.org/nu/) can help you locate errors in your source or rendered markup. If any validation errors are found, they can then be corrected which will help decrease the likelihood of ATs incorrectly parsing your website.  
+  - Follow the official HTML specifications for guidance on writing correct markup. E.g. make sure to close open tags (`<div class="a-class">...</div>`), use unique `id`s, and don't incorrectly nest HTML elements.
+* [4.1.2 Name, role, value](https://www.w3.org/TR/WCAG21/#name-role-value) are related to the components and controls that make up the web interface. Specifically, that these components of the page can have their name and role programmatically determined, and that their states, properties and values can be set by the individual interacting with them.
```

Comment:
#### [4.1.2 Name, role, value](https://www.w3.org/TR/WCAG21/#name-role-value)
This rule is related to the components and controls that make up a web interface. It helps to ensure components on a page can have their name and role programmatically determined, and that their states, properties, and values can be set by the individual interacting with them.

Reply:
Matching the subhead edit we set above for Parsing.